README.md

```diff
@@ -96,12 +96,13 @@ We vendor some data from other repositories that you will sometimes need to upda
 
 **Note:** Deploys are mostly automated using `yarn release patch|minor|major`; the following instructions should be valid but take a look at [this issue](https://github.com/artsy/emission/issues/1077#issuecomment-401128949) if things break.
 
-**Note 2:** If you're updating React Native, you're gonna have to do a lot of this stuff manually. See the makefile/alloy/orta for more info.
+**Note 2:** If you're updating React Native, you're going to have to do a lot of this stuff manually. See the makefile or alloy for more info.
 
 1. Don't change the `package.json` version.
-1. For non-beta releases, use either `yarn release patch`, `yarn release minor`, or `yarn release major`. For
-   beta releases use `yarn release prerelease`.
-1. If integrating in Eigen consult the docs [on updating Emission](https://github.com/artsy/eigen/blob/master/docs/updating_emission.md)
+2. For non-pre-releases, use either `yarn release patch`, `yarn release minor`, or `yarn release major`. For
+   pre-releases, use `yarn release prerelease`. Pre-releases are usually reserved for testing major releases before deployment.
+3. After the release has finished compiling, a PR will be automatically created in Metaphysics, which can be self assigned and merged.
```

Comment:
Maybe add some detail about what the PR contains.

Reply:
👍